modules/gapi/src/executor/gasync.cpp

```diff
@@ -19,11 +20,11 @@
 namespace {
     //This is a tool to move initialize captures of a lambda in C++11
     template<typename T>
-    struct move_through_copy{
+    struct copy_through_move{
```

Comment:
Is it somehow related to cancelation or just naming fix?

Reply:
not related at all